boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -103,7 +117,9 @@ if (CONFIG_BUILD_WITH_TFM)
 
 		set_property(GLOBAL APPEND PROPERTY extra_post_build_byproducts
 			${CMAKE_BINARY_DIR}/tfm_s_signed.bin
+			${CMAKE_BINARY_DIR}/tfm_s_signed.hex
```

Comment:
Can you append these commands to the `extra_post_build_byproducts` in a separate cmake statement?

To avoid merge conflicts when upmerging.